packages/mdx-deck-code-surfer/src/deck-code-surfer.js

```diff
@@ -1,93 +1,76 @@
 import CodeSurfer from "code-surfer";
 import React from "react";
-import { modes, incStep, decStep } from "mdx-deck";
-import { withDeck } from "mdx-deck/dist/context";
-import { withSlide } from "mdx-deck/dist/Slide";
+import { withDeck, withSlide } from "mdx-deck";
+import { setSteps } from "mdx-deck/dist/updaters";
+import { modes } from "mdx-deck/dist/constants";
```

Comment:
Looks like you don't need to use the `modes` constants anymore – ideally anything you need should be available without digging into the `dist/` path. I can expose the updaters as part of the public API as well

Reply:
I'm not using `modes` right now, but I'll probably need some way of knowing which one is the current mode later.

Reply:
yeah, I'm thinking internally there should be a way to know whether components like this are "live" (i.e. being presented) vs displaying all contents, such as when exporting to PDF. I can expose the updaters and constants from the main `index.js` for now, but would also like to have a simpler API for third party components to be able to hook into